app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.java

```diff
@@ -221,19 +222,5 @@ protected void onProfileClicked() {
     protected void onPeerReviewClicked() {
         ReviewActivity.startYourself(getActivity(), getString(R.string.title_activity_review));
     }
-
-    private class BaseLogoutListener implements CommonsApplication.LogoutListener {
```

Comment:
There are some unused imports as a consequence of this change. Kindly remove them too.